brands/amenity/cafe.json

```diff
@@ -761,6 +761,23 @@
       "takeaway": "yes"
     }
   },
+  "amenity/cafe|Juan Valdez Café": {
+    "locationSet": {"include": ["001"]},
+    "tags": {
+      "amenity": "cafe",
+      "brand": "Juan Valdez Café",
+      "brand:wikidata": "Q6301102",
+      "brand:wikipedia": "en:Juan Valdez Café",
+      "cuisine": "coffee_shop;empanada;biscuit",
+      "internet_access": "wlan",
```

Comment:
Are you sure that it is true for every single cafe using this brand, worldwide?

Reply:
Yup. Can find the url for the franchise requirements right now but, to the best of my knowledge (because I worked at one), they DO require items on the menu, services, wifi, and outdoor seating (unless not possible), even worldwide (recipes for empanadas are provided)

Reply:
I don't see anything about empanadas on their website.  It just looks like a normal coffeehouse to me, and it should probably have similar tags to Starbucks.

https://www.juanvaldezcafe.com/en-us/menu/

Reply:
I see one as a thumbnail here https://www.juanvaldezcafe.com/en-us/snacks/

Reply:
If you click on the image of the empanadas it's not including in the list of salty snakes. Whereas, if you click on the picture of chocolate cake it is listed with the other sweet treats. Which makes me think they don't serve empanadas despite the image. 

Anyway, it veers into "every blade of grass" mapping to include stuff like that as it is. Since it's not their main/secondary/or even third thing if they even do serve it. Otherwise, why not also list cinnamon rolls, chocolate cake, brownies, muffins, etc etc? If they do serve them, there's nothing particularly special about empanadas compared to anything else on the menu. Same goes for biscuits. So, both should be left out IMO. Otherwise, people will think it's an "empanadas and biscuit cafe." Which it isn't.

Reply:
OK, updated the cuisine tag